packages/theme-chalk/src/radio.scss

```diff
@@ -6,7 +6,7 @@
 @import 'mixins/button';
 @import 'common/var';
 
-:root {
+@include b(radio) {
```

Comment:
found a typo in line#15

Also can you move this into body after line#14?

Reply:
Fixed.